app/uk.gov.hmrc.helptosavestridefrontend/views/account_already_exists.scala.html

```diff
@@ -17,32 +17,51 @@
 @import uk.gov.hmrc.helptosavestridefrontend.config.FrontendAppConfig
 @import uk.gov.hmrc.helptosavestridefrontend.controllers.routes
 
-@this(main: main_template)
+@import uk.gov.hmrc.helptosavestridefrontend.views.html.layout
+@import uk.gov.hmrc.helptosavestridefrontend.views.html.partials._
 
-@(accountNumber: Option[String])(implicit messages: Messages, appConfig: FrontendAppConfig)
+@this(
+  layout: layout,
+  h1: h1,
+  p: paragraph,
+  a: link,
+  button: button,
+  ul: ul
+)
 
-@title = @{Messages("hts.account.exists.title.h1")}
+@(
+  accountNumber: Option[String]
+)(
+  implicit request: Request[_],
+  config: FrontendAppConfig,
+  messages: Messages
+)
 
-@main(
- title = title,
- bodyClassesOption = None) {
+@layout(title = messages("hts.account.exists.title.h1")) {
 
- <h1 class="heading-large">@title</h1>
+  @h1(messages("hts.account.exists.title.h1"))
 
- <p>@messages("hts.account.exists.p1")</p>
- @{accountNumber.map { number ⇒
-  <p>{messages("hts.payment-details.details-are")}</p>
+  @p(messages("hts.account.exists.p1"))
 
-  <ul class="list list-bullet">
-   <li>{messages("hts.payment-details.sort-code.text")} <span class="bold">{appConfig.NsiBankTransferDetails.sortCode.replace(" ", "").grouped(2).mkString(" ")}</span></li>
-   <li>{messages("hts.payment-details.account-number.text")} <span class="bold">{appConfig.NsiBankTransferDetails.accountNumber}</span></li>
-   <li>{messages("hts.payment-details.payment-reference-number.text")} <span class="bold">{number}</span></li>
-  </ul>
+  @p(messages("hts.payment-details.details-are"))
 
-  <p>{messages("hts.payment-details.monthly-deadline.message")}</p>
- }}
+  @ul(items = Seq(
+    s"""${messages("hts.payment-details.sort-code.text")} <strong>${config.NsiBankTransferDetails.sortCode.replace(" ", "").grouped(2).mkString(" ")}</strong>""",
+    s"""${messages("hts.payment-details.account-number.text")} <strong>${config.NsiBankTransferDetails.accountNumber}</strong>""",
+    s"""${messages("hts.payment-details.payment-reference-number.text")} <strong>${accountNumber}</strong>"""
```

Comment:
because accountNumber is an Option[String] it may be empty, the original template mapped into it to get any value for printing

Reply:
done